Compilation of sdk-1.3.236.0 fails with gcc 5

I've tried to add this version to conan and compilation fails with gcc 5. Do you know minimum gcc version now?

```
Scanning dependencies of target spirv-cross-reflect
Scanning dependencies of target spirv-cross-core
[  4%] Building CXX object source_subfolder/CMakeFiles/spirv-cross-reflect.dir/spirv_reflect.cpp.o
[  9%] Building CXX object source_subfolder/CMakeFiles/spirv-cross-core.dir/spirv_cross.cpp.o
[ 14%] Building CXX object source_subfolder/CMakeFiles/spirv-cross-core.dir/spirv_parser.cpp.o
[ 19%] Linking CXX static library ../lib/libspirv-cross-reflect.a
[ 19%] Built target spirv-cross-reflect
[ 23%] Building CXX object source_subfolder/CMakeFiles/spirv-cross-core.dir/spirv_cross_parsed_ir.cpp.o
[ 28%] Building CXX object source_subfolder/CMakeFiles/spirv-cross-core.dir/spirv_cfg.cpp.o
[ 33%] Linking CXX static library ../lib/libspirv-cross-core.a
[ 33%] Built target spirv-cross-core
Scanning dependencies of target spirv-cross-util
Scanning dependencies of target spirv-cross-glsl
[ 38%] Building CXX object source_subfolder/CMakeFiles/spirv-cross-util.dir/spirv_cross_util.cpp.o
[ 42%] Building CXX object source_subfolder/CMakeFiles/spirv-cross-glsl.dir/spirv_glsl.cpp.o
/home/conan/w/prod/BuildSingleReference/.conan/data/spirv-cross/1.3.236.0/_/_/build/835c9ea83350713ab748cebe499e1c41efebcc63/source_subfolder/spirv_glsl.cpp: In member function â€˜spirv_cross::SmallVector<spirv_cross::TypedID<(spirv_cross::Types)3u> > spirv_cross::CompilerGLSL::get_composite_constant_ids(spirv_cross::ConstantID)â€™:
/home/conan/w/prod/BuildSingleReference/.conan/data/spirv-cross/1.3.236.0/_/_/build/835c9ea83350713ab748cebe499e1c41efebcc63/source_subfolder/spirv_glsl.cpp:4958:23: error: could not convert â€˜(spirv_cross::ID*)(& constant->spirv_cross::SPIRConstant::m.spirv_cross::SPIRConstant::ConstantMatrix::id)â€™ from â€˜spirv_cross::ID* {aka spirv_cross::TypedID<(spirv_cross::Types)0u>*}â€™ to â€˜spirv_cross::SmallVector<spirv_cross::TypedID<(spirv_cross::Types)3u> >â€™
    return constant->m.id;
                       ^
/home/conan/w/prod/BuildSingleReference/.conan/data/spirv-cross/1.3.236.0/_/_/build/835c9ea83350713ab748cebe499e1c41efebcc63/source_subfolder/spirv_glsl.cpp:4960:28: error: could not convert â€˜(spirv_cross::ID*)(& constant->spirv_cross::SPIRConstant::m.spirv_cross::SPIRConstant::ConstantMatrix::c[0].spirv_cross::SPIRConstant::ConstantVector::id)â€™ from â€˜spirv_cross::ID* {aka spirv_cross::TypedID<(spirv_cross::Types)0u>*}â€™ to â€˜spirv_cross::SmallVector<spirv_cross::TypedID<(spirv_cross::Types)3u> >â€™
    return constant->m.c[0].id;
```

--- a/spirv_cross_containers.hpp
+++ b/spirv_cross_containers.hpp
@@ -226,7 +226,7 @@ public:
 	}
 
 	template <typename U, size_t M>
-	SmallVector(const U (&init)[M]) SPIRV_CROSS_NOEXCEPT : SmallVector(init, init + M)
+	explicit SmallVector(const U (&init)[M]) SPIRV_CROSS_NOEXCEPT : SmallVector(init, init + M)
 	{
 	}
 

--- a/spirv_glsl.cpp
+++ b/spirv_glsl.cpp
@@ -4965,9 +4965,9 @@ SmallVector<ConstantID> CompilerGLSL::get_composite_constant_ids(ConstantID cons
 		if (is_array(type) || type.basetype == SPIRType::Struct)
 			return constant->subconstants;
 		if (is_matrix(type))
-			return constant->m.id;
+			return SmallVector<ConstantID>(constant->m.id);
 		if (is_vector(type))
-			return constant->m.c[0].id;
+			return SmallVector<ConstantID>(constant->m.c[0].id);
 		SPIRV_CROSS_THROW("Unexpected scalar constant!");
 	}
 	if (!const_composite_insert_ids.count(const_id))

